eisbuk-admin/src/components/atoms/UserAttendance/__testData__/dummyData.ts

```diff
@@ -0,0 +1,9 @@
+import { Category } from "eisbuk-shared";
+
+export const Saul = {
+  name: "Saul",
+  surname: "Goodman",
+  certificateExpiration: "2001-01-01",
+  customer_id: "saul",
+  category: Category.PreCompetitive,
+};
```

Comment:
I like this :+1: , however you should write tests against in the parent component in this case.